AWSAuthSDK/Sources/AWSAuthCore/AWSAuthUIHelper.h

```diff
@@ -69,6 +80,23 @@ NS_ASSUME_NONNULL_BEGIN
  **/
 + (BOOL) isBackgroundColorFullScreen:(id<AWSUIConfiguration>)config;
 
+/**
+ Check if a `UIColor` is bright or dark. Based on http://www.w3.org/WAI/ER/WD-AERT/#color-contrast
+ */
++ (BOOL) isDarkColor:(UIColor *) color;
```

Comment:
Is this something we want to expose as a public method in the helper class? I can see its value for our internal implementation, but I'm reluctant to be in the business of providing styling advice in an AuthUIHelper class. :)